packages/-ember-data/tests/unit/store/serializer-for-test.js

```diff
@@ -29,23 +30,37 @@ module('unit/store/serializer_for - DS.Store#serializerFor', function(hooks) {
     );
   });
 
-  test('Calling serializerFor with a type that has not been registered looks up the default ApplicationSerializer', function(assert) {
-    const ApplicationSerializer = DS.JSONSerializer.extend();
+  deprecatedTest(
```

Comment:
this test should not be deprecated because it is using the application serializer fallback